.github/ISSUE_TEMPLATE/release-template.md

```diff
@@ -16,7 +16,7 @@ git checkout origin/HEAD
 git switch --create release/$RELEASE
 git push origin release/$RELEASE
 ```
-
+- [ ] create a release branch for repo [harvest-metrics](https://github.com/NetApp/harvest-metrics) with the name specified in the previous step.
```

Comment:
- [ ] Create a release branch for the [harvest-metrics](https://github.com/NetApp/harvest-metrics) repo like so:
```bash
RELEASE=23.02.0
git clone https://github.com/NetApp/harvest-metrics.git
git checkout origin/HEAD
git switch --create release/$RELEASE
git push origin release/$RELEASE
```